display_list/BUILD.gn

```diff
@@ -201,10 +202,6 @@ if (enable_unittests) {
   }
 }
 
-fixtures_location("display_list_benchmarks_fixtures") {
```

Comment:
Is this really unused?

Reply:
I did not see any references to fixtures in the DL benchmarks.  The benchmarks appear to be running as expected with this change.